buffer/src/main/java/io/netty/buffer/AdaptivePoolingAllocator.java

```diff
@@ -112,6 +111,13 @@ enum MagazineCaching {
     private static final int CENTRAL_QUEUE_CAPACITY = SystemPropertyUtil.getInt(
             "io.netty.allocator.centralQueueCapacity", NettyRuntime.availableProcessors());
 
+    /**
+     * The capacity if the magazine local buffer queue. This queue just pools the outer ByteBuf instance and not
+     * the actual memory and so helps to reduce GC pressure.
+     */
+    private static final int MAGAZINE_BUFFER_QUEUE_CAPACITY = SystemPropertyUtil.getInt(
+            "io.netty.allocator.magazineBufferQueueCapacity", 256);
```

Comment:
4096 felt like too many… I don't know if this is too few 😅

Reply:
Maybe just do 1024 for now ? In reality there shouldn't be too many magazines.